src/Http/Requests/UpdateFormBuilderSectionRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace FluxErp\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateFormBuilderSectionRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'id' => 'required|exists:form_builder_sections,id,deleted_at,NULL',
+            'form_id' => 'required|exists:form_builder_forms,id,deleted_at,NULL',
```

Comment:
Do you really want to make the form_id column updateable?

Reply:
nope